<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCAI_Writer</class>
 <widget class="QWidget" name="CCAI_Writer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_info">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>info</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTextBrowser" name="textBrowser_info"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ccai_writer">
     <property name="title">
      <string>文章生成器</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QToolBox" name="toolBox_ccai_writer">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <widget class="QWidget" name="page_ccai_writer_1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>354</width>
           <height>272</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第一步：输入文章主题</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="5" column="0" colspan="2">
           <widget class="QTextEdit" name="textEdit_ccai_writer_1_content"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_ccai_writer_1_content">
            <property name="text">
             <string>内容:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_ccai_writer_1_title">
            <property name="text">
             <string>标题:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_1_random">
            <property name="text">
             <string>随机主题</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_1_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QTextEdit" name="textEdit_ccai_writer_1_title">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>354</width>
           <height>249</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第二步：配置用户自定义信息</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_ccai_writer_2">
            <property name="text">
             <string>文章中需要存在的自定义内容</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTextEdit" name="textEdit_ccai_writer_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_2_default">
            <property name="text">
             <string>默认</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_2_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>354</width>
           <height>249</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第三步：获取AI输出并优化内容</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_ccai_writer_3">
            <property name="text">
             <string>优化建议</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_3_regenerate">
            <property name="text">
             <string>重新生成</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_3_open">
            <property name="text">
             <string>打开预览文件</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_ccai_writer_3_optimize">
            <property name="text">
             <string>输入优化</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pushButton_ccai_writer_3_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QTextEdit" name="textEdit_ccai_writer_3_optimize"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>337</width>
           <height>238</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第四步：生成markdown格式</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" colspan="2">
           <widget class="QTabWidget" name="tabWidget_ccai_writer_4">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_ccai_writer_4_base">
             <attribute name="title">
              <string>基础配置</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_ccai_writer_4_base_path">
                <property name="text">
                 <string>生成路径</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_writer_4_base_articleheader"/>
              </item>
              <item row="3" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_ccai_writer_4_base_catalogue"/>
              </item>
              <item row="2" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_ccai_writer_4_base_cover"/>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="pushButton_ccai_writer_4_base_path">
                <property name="text">
                 <string>选择路径</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1" colspan="2">
               <widget class="QComboBox" name="comboBox_writer_4_base_articlefooter"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_ccai_writer_4_base_path"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_ccai_writer_4_base_articlefooter">
                <property name="text">
                 <string>配置文章尾部</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_ccai_writer_4_base_articleheader">
                <property name="text">
                 <string>配置文章头部</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_ccai_writer_4_base_catalogue">
                <property name="text">
                 <string>目录配置</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_ccai_writer_4_base_cover">
                <property name="text">
                 <string>文章封面</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_ccai_writer_4_expand">
             <attribute name="title">
              <string>更多配置</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_4_default">
            <property name="text">
             <string>默认配置</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_4_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>354</width>
           <height>253</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第五步：生成xmind思维导图</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label_ccai_writer_5_path">
            <property name="text">
             <string>生成路径</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QTextBrowser" name="textBrowser_ccai_writer_5">
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;111&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_ccai_writer_5_path">
            <property name="text">
             <string>选择路径</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pushButton_ccai_writer_5_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLineEdit" name="lineEdit_ccai_writer_5_path"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_5_skip">
            <property name="text">
             <string>跳过</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButton_ccai_writer_5_generate">
            <property name="text">
             <string>生成</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>354</width>
           <height>272</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第六步：效果呈现</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_6_open">
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QTextBrowser" name="textBrowser_ccai_writer_6"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_ccai_writer_6_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_6_regenerate">
            <property name="text">
             <string>重新生成</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>337</width>
           <height>252</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第七步：继续优化</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_ccai_writer_7">
            <property name="title">
             <string>配置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="2" column="3" colspan="3">
              <widget class="QComboBox" name="comboBox_ccai_writer_7_xmind">
               <item>
                <property name="text">
                 <string>思维导图</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>无</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="3" colspan="3">
              <widget class="QComboBox" name="comboBox_ccai_writer_7_catalogue">
               <item>
                <property name="text">
                 <string>目录</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>无目录</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="3" colspan="3">
              <widget class="QComboBox" name="comboBox_ccai_writer_7_usercontent">
               <item>
                <property name="text">
                 <string>用户自定义</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>无</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="5" column="3" colspan="3">
              <widget class="QComboBox" name="comboBox_ccai_writer_7_article">
               <item>
                <property name="text">
                 <string>文章</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>无</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_ccai_writer_7">
               <property name="text">
                <string>文章结构</string>
               </property>
              </widget>
             </item>
             <item row="3" column="3" colspan="3">
              <widget class="QComboBox" name="comboBox_ccai_writer_7_articleheader">
               <item>
                <property name="text">
                 <string>文首</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>无</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="6" column="3" colspan="3">
              <widget class="QComboBox" name="comboBox_ccai_writer_7_articlefooter">
               <item>
                <property name="text">
                 <string>文末</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>无</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="6">
              <spacer name="horizontalSpacer_ccai_writer_7">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_7_back">
            <property name="text">
             <string>上一步</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_7_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>354</width>
           <height>232</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第八步：输出markdown文件</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="0">
           <widget class="QLabel" name="label_ccai_writer_8_path">
            <property name="text">
             <string>最终生成路径</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_ccai_writer_8_path"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_ccai_writer_8_path">
            <property name="text">
             <string>选择路径</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_8_back">
            <property name="text">
             <string>上一步</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_ccai_writer_8_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_9">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>354</width>
           <height>232</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第九步：选择发布平台</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QComboBox" name="comboBox_ccai_writer_9_platform">
            <item>
             <property name="text">
              <string>知乎</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CSDN</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>公众号</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_9_platform">
            <property name="text">
             <string>去发布</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_9_back">
            <property name="text">
             <string>上一步</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_9_confirm">
            <property name="text">
             <string>确认</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_ccai_writer_10">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>354</width>
           <height>238</height>
          </rect>
         </property>
         <attribute name="label">
          <string>第十步：结束</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_ccai_writer_10_retry">
            <property name="text">
             <string>再来一篇</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_ccai_writer_10_confirm">
            <property name="text">
             <string>完成</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTextBrowser" name="textBrowser_ccai_writer_10"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
